storage/tokudb/hatoku_defines.h

```diff
@@ -60,6 +60,9 @@ Copyright (c) 2006, 2015, Percona and/or its affiliates. All rights reserved.
 #include "misc.h"
 #endif
 
+#include <string>
```

Comment:
I guess there are reasons why not ha_tokudb.h itself includes these?

Reply:
Yeah, geneally Toku tries to have all 'outside' includes in hatoku_defines.h, particularly anything mysql (which this is admittedly not) to make sure they are in proper order as there are problems with including various things from the server if not done in exactly the correct order :(